assignProdTaskChain.py

```diff
@@ -154,12 +149,24 @@ def assignRequest(url, workflow, team, site, era, procstr, procver, activity, lf
         print res
 
 def getRequestDict(url, workflow):
-    conn = httplib.HTTPSConnection(url, cert_file=os.getenv(
-        'X509_USER_PROXY'), key_file=os.getenv('X509_USER_PROXY'))
-    r1 = conn.request("GET", '/reqmgr/reqMgr/request?requestName=' + workflow)
+    headers = {"Accept": "application/json"}
+    conn = httplib.HTTPSConnection(url, cert_file=os.getenv('X509_USER_PROXY'), key_file=os.getenv('X509_USER_PROXY'))
+    urn = "/reqmgr2/data/request/%s" % workflow
+    conn.request("GET", urn, headers=headers)
     r2 = conn.getresponse()
-    request = json.loads(r2.read())
-    return request
+    request = json.loads(r2.read())["result"][0]
+    return request[workflow]
+
+def findOriginalProcessingString(url,workflow):
```

Comment:
And this method is tricky. You have to return a (or two) dictionary, like:
```
{"AcquisitionEra": {tempSchema['Task1']['TaskName']: tempSchema['Task1']['AcquisitionEra'],
                              tempSchema['Task2']['TaskName']: tempSchema['Task2']['AcquisitionEra'],
                              etc}
 "ProcessingString": {tempSchema['Task1']['TaskName']: tempSchema['Task1']['ProcessingString'],
                              tempSchema['Task2']['TaskName']: tempSchema['Task2']['ProcessingString'],
                              etc}
}
```
and then you modify you return and where you call this function according.